apps/sasquatch-kotlin/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Ideally we just migrate the sasquatch little by little. Maintaining 2 apps is not what we want.